### PR TITLE
fixed -> clicking on commit link on /feed. redirects to GitHub API

### PR DESCRIPTION
**Pull request by manu-r12**
The actual problem was that the url to specifc commit was taking to the GitHub API not to the actual html url

### CODE REVIEW

File: components/DateRangePicker.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React from "react";
+`import React from "react";
```
Comment (rithviknishad) [inline components/DateRangePicker.tsx:1]: I assume this change was accidental?

```suggestion
import React from "react";
```

File: yarn.lock
Comment (rithviknishad) [inline yarn.lock:1]: Remove this file, we are using pnpm lock file in this repo.

**Review by rithviknishad — CHANGES_REQUESTED** [review]
Deploy failing...

image

File: yarn.lock
Comment (manu-r12) [reply yarn.lock:1]: > Remove this file, we are using pnpm lock file in this repo.

which one?

Comment (manu-r12) [reply yarn.lock:1]: Done ✅

**Comment by manu-r12** [comment]
@rithviknishad  can you review it now 😃

**Review by rithviknishad — APPROVED** [review]

**Pull request merged**